openstudiocore/src/cli/openstudio_cli.rb

```diff
@@ -311,6 +312,9 @@ def execute(sub_argv)
       o.on('-p', '--postprocess_only', 'Only run the reporting measures') do
         options[:post_process] = true
       end
+      o.on('-s', '--socket PORT', 'Pipe status messages to a socket on localhost PORT') do |port|
+        options[:socket] = port
+      end      
```

Comment:
seems like we can upgrade this as a web socket with a configurable address and get a good bit of mileage out of this. someday. This will do for App needs.